eva-server/src/main/resources/application.properties

```diff
@@ -17,4 +17,6 @@ spring.jmx.enabled = false
 springfox.documentation.swagger.v2.path=/webservices/api
 
 #spring.profiles.active=oauth2-security
-#security.oauth2.resource.user-info-uri = ...
\ No newline at end of file
+#security.oauth2.resource.user-info-uri = ...
+
+eva.mongo.collections.files=@eva.mongo.collections.files@
```

Comment:
then yes. I think some work will have to be done on MultiMongoFactoryConfiguration (because it currently also reads from eva.properties), but I don't see big reasons against that.